### PR TITLE
Fix/use hakyll config

### PR DESCRIPTION
**Pull request by Maeevick**
https://github.com/haskellfoundation/haskellfoundation.github.io/issues/120

### CODE REVIEW

**Comment by Maeevick** [comment]
@myShoggoth I failed my push and the target 🎯 branch but it should be ok now, I don't know if this PR is relevant know but it can be a reminder 😉

File: site.hs [hunk 1]
```diff
@@ -9,12 +9,10 @@ import Data.List (sortBy)
...
 
 config :: Configuration
-config = defaultConfiguration
- { destinationDirectory = "../docs"
- }
+config = defaultConfiguration { destinationDirectory = "docs" }
 
 main :: IO ()
-main = hakyll $ do
+main = hakyllWith config $ do
```
Comment (elland) [inline site.hs:15]: Good catch :)

**Review by elland — APPROVED** [review]
Nice!

File: site.hs (hunk 1 above)
Comment (Maeevick) [reply site.hs:15]: I will rebase tonight (GMT+2 🇫🇷)

**Comment by myShoggoth** [comment]
@Maeevick if you fix the merge conflict I'll merge the PR.

**Comment by Maeevick** [comment]
> @Maeevick if you fix the merge conflict I'll merge the PR.

@myShoggoth will be done in few hours when baby is going to sleep ☺️

**Comment by Maeevick** [comment]
@myShoggoth @elland I sucked with my branching and pull request, so the good one is here (rebased on hakyll and merge conflicts solved) https://github.com/haskellfoundation/haskellfoundation.github.io/pull/122

**Pull request closed without merging**